stories/MenuBar.stories.tsx

```diff
@@ -1,18 +1,100 @@
 import type { Meta, StoryObj } from '@storybook/react';
-import type { ComponentProps } from 'react';
+import { type ComponentProps, useState } from 'react';
+import { Section, Stack } from '../lib/components';
 import { MenuBar } from '../lib/components/MenuBar';
 
-type StoryProps = ComponentProps<typeof MenuBar>;
+type StoryProps = ComponentProps<typeof MenuBar.Dropdown>;
 
 export default {
-  component: MenuBar,
-  title: 'Components/MenuBar',
+  component: MenuBar.Dropdown,
+  subcomponents: {
+    Menubar: MenuBar,
+    'MenuBar.Dropdown.MenuItem': MenuBar.Dropdown.MenuItem,
+    'MenuBar.Dropdown.MenuItemToggle': MenuBar.Dropdown.MenuItemToggle,
+  },
+  title: 'Components/MenuBar.Dropdown',
+  decorators: (Story) => (
+    <Stack direction="column">
+      <Stack.Item>
+        <Story />
+      </Stack.Item>
+      <Stack.Item>
+        <Section title="Foo">Bar</Section>
+      </Stack.Item>
+    </Stack>
+  ),
 } satisfies Meta<StoryProps>;
 
-type Story = StoryObj<StoryProps>;
-
-export const Default: Story = {
+export const Default: StoryObj<StoryProps> = {
   args: {
-    children: 'MenuBar',
+    disabled: false,
+    openWidth: '22rem',
+  },
+  render: (args) => {
+    const [openMenuBar, setOpenMenuBar] = useState<string | null>(null);
+    const [openOnHover, setOpenOnHover] = useState<boolean>(false);
+    const [checkbox, setCheckbox] = useState<boolean>(false);
```

Comment:
no need to cast on things with a valid initial value like this. 
`useState(false)` pls